pkg/roachprod/install/nodes.go

```diff
@@ -44,7 +44,9 @@ func ListNodes(s string, numNodesInCluster int) (Nodes, error) {
 		return nil, errors.AssertionFailedf("invalid number of nodes %d", numNodesInCluster)
 	}
 
-	if s == "all" {
+	// "lb" is a special value that also returns all nodes, but implies a load
+	// balancer should be used when applicable.
```

Comment:
Why "when applicable"? I don't see logic below to handle the case where a load balancer is not available (would we want that?)

Reply:
True, I'll update the comment to be more concise.